.github/workflows/deploy.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy blog
```

Comment:
```suggestion
name: Build blog
```

Protože deploy je jen někdy, ale build vždycky?

Reply:
Hmm, vždycky je jen lint, nevím jestli Danube Delta umí zvlášť build.

Reply:
Hmm, vidím jenom preview, ale to se bude snažit otevřít prohlížeč si myslím https://github.com/honzajavorek/danube-delta/#usage

Reply:
Každopádně build je vždycky, protože když neprojde tak zčervená pull request. Deploy je jen pro začleněné změny.